lib/api.js

```diff
@@ -32,6 +32,10 @@ api.declare({
   scopes:      [['secrets:set:<name>']],
   title:       'Set Secret',
   stability:    'stable',
+  cleanPayload: payload => {
```

Comment:
Not necessarily, since some payloads don't include a secret -- that might confuse the user!